wBoooksApi/.idea/.generators

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Can we add the entire .idea folder to the git ignore? Remember to remove these files after that